zpeclient.go

```diff
@@ -172,11 +175,17 @@ func (c *Cache) parseData(item *v1.AthenzDomain) (roleMappings, error) {
 				action:   actionRegex,
 				effect:   effect,
 			}
-			_, ok := crMap.roleToAssertion[assertion.Role]
-			if !ok {
-				crMap.roleToAssertion[assertion.Role] = []*simpleAssertion{}
+			if *assertion.Effect != 0 && *assertion.Effect == zms.DENY {
```

Comment:
Why do we need to check if the assertion is not equal to 0? The second check for the zms DENY should be enough here.

Reply:
There can be a case that `*assertion.Effect` is null, we need to cover it as well.

Reply:
We already have the check here to see if effect is not nil: https://github.com/yahoo/k8s-athenz-webhook/blob/master/zpeclient.go#L155 
The if statement here is dereferencing the actual value of assertion.Effect meaning that you're checking against the int value 0, not the nil pointer so this can actually be removed.

Reply:
Can we also reuse the effect variable here defined previously?